views/beneficial-owner-type.html

```diff
@@ -103,27 +103,15 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">{{ boTypesTitle }}</h1>
           items: boTypes
         }) }}
 
-        <button class="govuk-button" data-module="govuk-button" id="add">Add<span class="govuk-visually-hidden"> {{ whatToAdd }} </span></button>
+        <button class="govuk-button" data-module="govuk-button" id="add" formaction="/register-an-overseas-entity/beneficial-owner-type" >Add<span class="govuk-visually-hidden"> {{ whatToAdd }} </span></button>
```

Comment:
Just as the `submitUrl` is passed in to the template as a param, I think this URL should be passed in too (or, we pass neither in). Needs to be a consistent approach though. I think passing the URLs in is the right way to go though.

Reply:
Tried that and it won't work because the submit version of the page url does not have a get, so there is nothing to pass that value in and so there is only one way to guarantee that this original url will be present for when press that add button in cases where we have tried to press no more to add and an error has been thrown with the "/submit" url.

Reply:
Not only is there no get for "/submit" but there is not point where we would be able to use a get, it is fundamentally as POST url.

Reply:
Can't understand why this is not possible. I will check the branch out and investigate.

Reply:
All clear now. A quirk of using 'formAction' and the validation middleware that doesn't add back in params when page is rerendered.

Agreed with @markpit that we'll hard-code both the URLs for consistency.